network_observability/observing-network-traffic.adoc

```diff
@@ -17,18 +17,15 @@ include::modules/network-observability-pktdrop-overview.adoc[leveloffset=+2]
 [role="_additional-resources"]
 .Additional resources
 * xref:../network_observability/observing-network-traffic.adoc#network-observability-packet-drops_nw-observe-network-traffic[Working with packet drops]
+* xref:../network_observability/metrics-alerts-dashboards.adoc#network-observability-metrics_metrics-dashboards-alerts[Viewing Network Observability metrics dashboards]
```

Comment:
This link text should probably be Network Observability metrics, as in [below](https://github.com/openshift/openshift-docs/pull/69106/files#diff-21cba4b9eea6286345ee8e2ad7b3cc0e4dc6ac125a47623a7df9f52f2f2506aaR27).

```suggestion
* xref:../network_observability/metrics-alerts-dashboards.adoc#network-observability-metrics_metrics-dashboards-alerts[Network Observability metrics]
```